sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -98,8 +98,9 @@ public TransportResult send(SentryEvent event) throws IOException {
 
     connection.connect();
 
-    try (OutputStream outputStream = connection.getOutputStream()) {
-      serializer.serialize(event, new OutputStreamWriter(outputStream, UTF_8));
+    try (OutputStream outputStream = connection.getOutputStream();
+        OutputStreamWriter outputStreamWriter = new OutputStreamWriter(outputStream, UTF_8)) {
```

Comment:
Why did the tool suggest we assign to a variable?
To help with identifying the line of a crash?
Looking at this just seems like more boiler plate.

Reply:
@bruno-garcia this way both streams get automatically closed/handled by try-with-resources feature